### PR TITLE
AUT-266_tpo_part2

### PR DESCRIPTION
**Pull request by azayatsfv**
GET get_tpo_roles
POST set_tpo_approval_queue_data

### CODE REVIEW

**Review by darkhipenko — APPROVED** [review]
Approved

**Comment by azayatsfv** [comment]
I have udpated POST set_tpo_approval_queue_data. @darkhipenko Please take a look and make an approve if everything is correct.

File: APIs/tpo/requests/set_tpo_approval_queue_data.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    def build_payload(self) -> Dict[str, List[Dict[str, Any]]]:
+        payload_list = []
+        for payload_key in self.attr_list:
+            if getattr(self, payload_key.lower(), None) is not None:
+                payload_list.append(
+                    {DataKeys.FIELD_NAME: getattr(self.data_payload, payload_key.upper(), payload_key),
+                     "FieldNewValue": getattr(self, payload_key.lower())})
+        payload = {self.REQUEST_PAYLOAD_KEY: payload_list}
+        return payload
```
Comment (darkhipenko) [inline APIs/tpo/requests/set_tpo_approval_queue_data.py:66]: Will be inherited from `KwargsRequestModel`, no need in this block of code

**Review by darkhipenko — CHANGES_REQUESTED** [review]
Please, review request file for `set_tpo_approval_qeue_data`

File: APIs/tpo/requests/set_tpo_approval_queue_data.py (hunk 1 above)
Comment (darkhipenko) [reply APIs/tpo/requests/set_tpo_approval_queue_data.py:66]: Resolved, a different key used for building dictionary.

**Review by darkhipenko — APPROVED** [review]
Approved

File: APIs/tpo/requests/set_tpo_approval_queue_data.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        args[SetTPOApprovalQueueDataRequestParams.RESET_LINK] = self.reset_link
+        return args
+
+    def build_payload(self) -> Dict[str, List[Dict[str, Any]]]:
+        payload_list = []
+        for payload_key in self.attr_list:
+            if getattr(self, payload_key.lower(), None) is not None:
+                payload_list.append(
+                    {DataKeys.FIELD_NAME: getattr(self.data_payload, payload_key.upper(), payload_key),
+                     "FieldNewValue": getattr(self, payload_key.lower())})
```
Comment (azayatsfv) [inline APIs/tpo/requests/set_tpo_approval_queue_data.py:64]: DataKey: "FieldNewValue" differ from DataKey ("FieldValue") that used in build_payload method in KwargsRequestModel class

**Pull request merged**